gateway/pom.xml

```diff
@@ -102,6 +102,11 @@
       <artifactId>spring-security-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.ldaptive</groupId>
+      <artifactId>ldaptive-netscape</artifactId>
+      <version>1.0</version>
+    </dependency>
```

Comment:
still being used ? grepping `org.ldaptive` in the PR does not reveal anything

Reply:
It's indeed required in spring-security-ldap, but then I don't get why it is not an explicit dependency from it:

* https://github.com/spring-projects/spring-security/blob/main/ldap/src/main/java/org/springframework/security/ldap/ppolicy/PasswordPolicyResponseControl.java#L23-L30
* https://github.com/spring-projects/spring-security/issues/3127

Maybe it is because the dependency is optional or so.

Reply:
Mind adding a comment in the pom.xml? we wouldn't know why this dependency is there otherwise.